docs/containers-registries.conf.5.md

```diff
@@ -30,14 +30,16 @@ as well as among different namespaces/repositories within a registry.
 Given an image name, a single `[[registry]]` TOML table is chosen based on its `prefix` field.
 
 `prefix`: A prefix of the user-specified image name, i.e. using one of the following formats:
-  - _host_[`:`_port_]
-  - _host_[`:`_port_]`/`_namespace_[`/`_namespace_…]
-  - _host_[`:`_port_]`/`_namespace_[`/`_namespace_…]`/`_repo_
-  - _host_[`:`_port_]`/`_namespace_[`/`_namespace_…]`/`_repo_(`:`_tag|`@`_digest_)
+  - [`*.`]_host_[`:`_port_]
+  - [`*.`]_host_[`:`_port_]`/`_namespace_[`/`_namespace_…]
+  - [`*.`]_host_[`:`_port_]`/`_namespace_[`/`_namespace_…]`/`_repo_
+  - [`*.`]_host_[`:`_port_]`/`_namespace_[`/`_namespace_…]`/`_repo_(`:`_tag|`@`_digest_)
 
-    The user-specified image name must start with the specified `prefix` (and continue
+    The user-specified image name can start with the specified `prefix` (and continue
```

Comment:
I think that "must" is still appropriate as it relates to the matching.  The registry in question get selected iff the user-specified image name starts with `prefix`.  "Can" introduces some space of interpretation.